Common/Brokerages/DefaultBrokerageModel.cs

```diff
@@ -155,6 +156,9 @@ public decimal GetLeverage(Security security)
                 case SecurityType.Cfd:
                     return 50m;
 
+                case SecurityType.Crypto:
+                    return 3m;
```

Comment:
Perhaps default leverage should be 1 as most exchanges/brokerages don't support leverage yet? Later we can add custom leverage by brokerage model.

Reply:
It's only really the US regulated exchanges that cannot offer leverage. But I see your point, we should not treat gdax behaviour as the default for crypto.